static/docs/commands-reference/status.md

```diff
@@ -6,12 +6,15 @@ cache and local files, or between the local cache and remote cache.
 ## Synopsis
 
 ```usage
-usage: dvc status [-h] [-q | -v] [-j JOBS] [--show-checksums]
-                  [-c] [-r REMOTE] [-a] [-T] [-d]
+usage: dvc status [-h] [-v] [-j JOBS] [--show-checksums] [-q] [-c] [-r REMOTE]
+                  [-a] [-T] [-d]
                   [targets [targets ...]]
 
+Show changed states, compare local cache and remote storage.
+documentation: https://man.dvc.org/status
+
 positional arguments:
-  targets               Limit the scope to these stage files.
+  targets               DVC files.
```

Comment:
let's keep the previous version, and actually we need to update DVC itself. Can you create a patch for DVC repo with this simple change?

Reply:
Okay, I will revert this to the original version. Okay sure will patch it in DVC as well.

Reply:
@shcheklein  I have created a pull request for this in DVC kindly review it. 

https://github.com/iterative/dvc/pull/2111